### ISSUE TITLE
Banana Cream Pie

### ISSUE THREAD

**Issue by Zyptor** [ExcessiveAmountsOfZombies/Croptopia#102]
Hello!

I get this error message in my Minecraft logs when starting up a world, Is there anything I can do to fix it?

Error message:
```
Parsing error loading recipe croptopia:banana_cream_pie
com.google.gson.JsonSyntaxException: Unknown item tag 'c:vanillas'
	at net.minecraft.class_1856.method_8107(class_1856.java:171)
	at net.minecraft.class_1856.method_8102(class_1856.java:147)
	at me.thonk.croptopia.recipe.DamageDurabilitySerializer.getIngredients(DamageDurabilitySerializer.java:36)
	at me.thonk.croptopia.recipe.DamageDurabilitySerializer.read(DamageDurabilitySerializer.java:21)
	at me.thonk.croptopia.recipe.DamageDurabilitySerializer.method_8121(DamageDurabilitySerializer.java:15)
	at net.minecraft.class_1863.method_17720(class_1863.java:123)
	at net.minecraft.class_1863.method_20705(class_1863.java:54)
	at net.minecraft.class_1863.method_18788(class_1863.java:35)
	at net.minecraft.class_4080.method_18790(class_4080.java:13)
	at net.minecraft.class_4080$$Lambda$5674/1505795402.accept(Unknown Source)
	at java.util.concurrent.CompletableFuture.uniAccept(CompletableFuture.java:656)
	at java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:632)
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:442)
	at net.minecraft.class_4014.method_18365(class_4014.java:71)
	at net.minecraft.class_4014$$Lambda$5900/1370468333.run(Unknown Source)
	at net.minecraft.class_1255.method_18859(class_1255.java:144)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:118)
	at net.minecraft.class_1255.method_18857(class_1255.java:127)
	at net.minecraft.class_310.method_29604(class_310.java:1958)
	at net.minecraft.class_310.method_29610(class_310.java:1808)
	at net.minecraft.class_310.method_29607(class_310.java:1752)
	at net.minecraft.class_525.method_2736(class_525.java:298)
	at net.minecraft.class_525.method_19922(class_525.java:238)
	at net.minecraft.class_525$$Lambda$6354/1044393015.onPress(Unknown Source)
	at net.minecraft.class_4185.method_25306(class_4185.java:33)
	at net.minecraft.class_4264.method_25348(class_4264.java:16)
	at net.minecraft.class_339.method_25402(class_339.java:149)
	at net.minecraft.class_4069.method_25402(class_4069.java:27)
	at net.minecraft.class_312.method_1611(class_312.java:92)
	at net.minecraft.class_312$$Lambda$6051/9353649.run(Unknown Source)
	at net.minecraft.class_437.method_25412(class_437.java:435)
	at net.minecraft.class_312.method_1601(class_312.java:92)
	at net.minecraft.class_312.method_22686(class_312.java:162)
	at net.minecraft.class_312$$Lambda$6048/220272993.run(Unknown Source)
	at net.minecraft.class_1255.execute(class_1255.java:94)
	at net.minecraft.class_312.method_22684(class_312.java:162)
	at net.minecraft.class_312$$Lambda$5322/338917009.invoke(Unknown Source)
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3174)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:127)
	at net.minecraft.class_310.method_1523(class_310.java:1071)
	at net.minecraft.class_310.method_1514(class_310.java:681)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
```

**Comment by Zyptor** [comment]
Ops, #67 already exists, my bad 😅

**Comment by smmmadden** [comment]
I'm running Croptopia on my MP server and see the same error with Croptopia v1.19.2-2.1.0
Was this fixed in an older version but missed in 1.19.x ?
![image](https://user-images.githubusercontent.com/10309346/196451090-5a160099-a9d4-4e9c-bbc3-7a3f1544a104.png)
